test/commands/force/data/record/dataRecord.nut.ts

```diff
@@ -247,4 +247,20 @@ describe('data:record commands', () => {
       expect(result).to.have.property('Bool__c', false);
     });
   });
+
+  it('will parse JSON correctly for update', () => {
```

Comment:
let's have a test to match the above comment--something that isn't valid json but contains curly brackets.